static/js/settings_exports.js

```diff
@@ -83,7 +73,7 @@ exports.set_up = function () {
         channel.post({
             url: '/json/export/realm',
             success: function () {
-                ui_report.success(i18n.t("Export started. Check back in a few minutes."), export_status);
+                ui_report.success(i18n.t("Export started. Check back in a few minutes."), export_status, 4000);
```

Comment:
I think this is a fine checkpoint, but I wonder if we can do even better by e.g. having a loading spinner in the "exports" table.